__init__.py

```diff
@@ -794,74 +815,87 @@ def handle_timer_stop(self, _):
         answer = self.ask_yesno(question)
         if answer == "yes":
             self._cancel_all_timers()
+            self._reset()
 
     def handle_wake_word_detected(self, _):
```

Comment:
Maybe this is a git formatting issue but it looks like this is a function with nothing in it except a comment. If this is indeed what I am seeing just remove the method completely.

Reply:
Must be a git formatting issue.  None of the methods in the skill class are empty..